code/dynamic_programming/no_consec_ones/no_consec_1.cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+using namespace std;
+
+int countNonConsecutiveOnes(int n)
+{
+    int endingZero[n], endingOne[n];
```

Comment:
Use vectors instead of arrays, and read the C++ Style Guide, (Although your code is pretty conformant).

Reply:
Updating code using vectors